docs/components/modeler/bpmn/timer-events/timer-events.md

```diff
@@ -81,6 +81,8 @@ A cycle defined as ISO 8601 repeating intervals format; it contains the duration
 
 - `R5/PT10S`: Every 10 seconds, up to five times
 - `R/P1D`: Every day, infinitely
+- `R/2022-01-01T10:00:00+02:00[Europe/Berlin]/P1D`: Every day infinitely, starting from the 1st of January 2022 at 10 am
+- `R5/2022-01-01T10:00:00+02:00[Europe/Berlin]/P1D`: Every day up to five times, starting from the 1st of January 2022 at 10 am
```

Comment:
👍 I like the `up to x times` wording.

🔧 It might be better to present somewhat different examples, e.g.:
- your first example repeating once per day at 10 am starting on the 1st of Jan 2022
- another example that repeats up to 3 times every 10 seconds starting at some specific date and time, e.g. `R3/2022-04-20T04:20:00Z/PT10S`

Reply:
🤔 Very peculiar chosen numbers 🌱